v2/tools/generator/internal/astmodel/type_visitor_test.go

```diff
@@ -236,6 +245,94 @@ func TestMakeTypeVisitorWithInjectedFunctions(t *testing.T) {
 			OptionalIntType,
 			"optional",
 		},
+		{
+			"EnumTypeHandler",
+			NewEnumType(StringType),
+			func(builder *TypeVisitorBuilder) {
+				builder.VisitEnumType = func(tv *TypeVisitor, ot *EnumType, _ interface{}) (Type, error) {
+					return ot, errors.New(ot.String())
+				}
+			},
+			NewEnumType(StringType),
+			"enum",
+		},
+		{
+			"EnumTypeSimplified",
+			NewEnumType(StringType),
+			func(builder *TypeVisitorBuilder) {
+				builder.VisitEnumType = func(mt *EnumType) (Type, error) {
+					return mt, errors.New(mt.String())
+				}
+			},
+			NewEnumType(StringType),
+			"enum",
+		},
+		{
+			"ObjectTypeHandler",
+			NewObjectType(),
+			func(builder *TypeVisitorBuilder) {
+				builder.VisitObjectType = func(tv *TypeVisitor, ot *ObjectType, _ interface{}) (Type, error) {
+					return ot, errors.New(ot.String())
+				}
+			},
+			NewObjectType(),
+			"object",
+		},
+		{
+			"ObjectTypeSimplified",
+			NewObjectType(),
+			func(builder *TypeVisitorBuilder) {
+				builder.VisitObjectType = func(mt *ObjectType) (Type, error) {
+					return mt, errors.New(mt.String())
+				}
+			},
+			NewObjectType(),
+			"object",
+		},
+		{
+			"OneOfTypeHandler",
+			NewOneOfType("x",BoolType,StringType),
```

Comment:
File needs to be formatted with `go fmt`.

Reply:
I suspect this file still hasn't been formatted - `go fmt` will add a single space after each `,`. 
I'll run CI but I expect it to fail because of this.